src/components/SearchCard.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { Card } from 'react-native-paper';
+
+export default function SearchCard({
+  treeName,
+  treeID,
```

Comment:
can you use the `Tree` type defined in .tsx?

Reply:
i thought it was best practice to pass individual props instead of whole objects (e.g. https://stackoverflow.com/questions/52621169/react-props-should-i-pass-the-object-or-its-properties-does-it-make-much-diffe)

Reply:
That seems like a remnant of JS, but since we're not using all the parts of `Tree` I don't see an issue

Reply:
Sure, lets pass individual props then, but declare the type outside of the prop signature and use lookup types:

```
import { Tree } from '...'

type SearchCardProps = {
    name: Tree["name"],
    id: Tree["id"]
}

export default function SearchCard(props: SearchCardProps)
const { name, id } = props;
```

the last part is a nit but I really dislike long / deconstructed signatures.